jackson-apt-processor/src/test/java/org/dominokit/jacksonapt/processor/BoxedTypesMapperTest.java

```diff
@@ -0,0 +1,69 @@
+package org.dominokit.jacksonapt.processor;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.File;
```

Comment:
Please the remove the unused imports

Reply:
Sure, thanks!

Reply:
I can remove only java.io.File. Both assertNotNull and assertEquals are used in the unit test.